packages/common/src/index.ts

```diff
@@ -0,0 +1,10 @@
+import BucketsConfig = require('./BucketsConfig');
+export { BucketsConfig };
```

Comment:
`export { BucketsConfig } from './BucketsConfig';` doesn't work?

Reply:
No, because `BucketsConfig.js` uses `export =`. It took me a while, but I tracked down the solution here: https://stackoverflow.com/questions/41892470/how-to-reexport-from-a-module-that-uses-export